examples/plot_simulate_alpha.py

```diff
@@ -19,21 +26,22 @@
 from hnn_core import simulate_dipole, read_params, Network
 
 ###############################################################################
-# Then we setup the directories and Neuron
+# Then we setup the directories and read the default parameters file
 hnn_core_root = op.dirname(hnn_core.__file__)
-
-###############################################################################
-# Then we read the default parameters file
 params_fname = op.join(hnn_core_root, 'param', 'default.json')
 params = read_params(params_fname)
 print(params)
 
 ###############################################################################
-# Update a few of the default parameters related to visualisation
+# Next we update a few of the default parameters related to visualisation. The
+# ``dipole_scalefctr`` relates to the amount of cortical tissue necessary to
```

Comment:
I wonder if we should instead add this line at the end:

```py
simulate_dipole(..., postproc=False)
dpl.scale(15000)
```

to discourage fiddling with `params` as much as possible.

This also makes me realize that `Dipole` is not documented in the API reference. Do you mind adding it there?

Reply:
+1!

I’d go as far as to stop reading any post-proc details from params (got bit in the rear by a dipole_smooth_win setting of 30, set to zero above). future PR...

Reply:
Good suggestion! I've removed all `params` edits except for tstop now,